src/components/common/dropdown.tsx

```
@@ -0,0 +1,109 @@
+    const handleClick = (event: MouseEvent) => {
+      // 트리거 영역 클릭 시 토글
+      if (
+        triggerRef.current &&
+        triggerRef.current.contains(event.target as Node)
+      ) {
+        toggleDropdown();
+        return;
+      }
+
+      // 드롭다운 내부 또는 외부 클릭 시 닫기
+      if (dropdownRef.current) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)